docs/src/pages/demos/breadcrumbs/RouterBreadcrumbs.js

```diff
@@ -106,7 +108,7 @@ class RouterBreadcrumbs extends React.Component {
               <List component="nav">
                 <ListItemLink to="/inbox" open={this.state.open} onClick={this.handleClick} />
                 <Collapse in={this.state.open} timeout="auto" unmountOnExit>
-                  <List component="div" disablePadding>
+                  <List component={'div'} disablePadding>
```

Comment:
Something we could improve in the TypeScript to JavaScript logic generation.

Reply:
I tried to change the js file directly, but I can't pass this ci. what should i do?
<img width="1032" alt="WX20190402-234503@2x" src="https://user-images.githubusercontent.com/26477346/55416471-6db22d80-55a1-11e9-97a4-2b0e4e3622ac.png">

Reply:
It's a topic for another pull request. I was only raising the potential.

Reply:
If it's fine with prettier it's fine with me